theme/templates/pages/group.html

```diff
@@ -860,13 +860,12 @@ <h4 class="modal-title" id="myModalLabel">Edit Group Details</h4>
             </div>
         </div>
     {% endif %}
-    </div>
+
 {% endblock %}
 
 {% block extra_js %}
     {{ block.super }}
-    <script type="text/javascript" src="{{ STATIC_URL }}js/vue-2.6.10-dev.js"></script>
     <script type="text/javascript" charset="utf8" src="{{ STATIC_URL }}js/jquery.dataTables.js"></script>
```

Comment:
Isn't this removal what broke the author filtering? Do you need to add it back?

Reply:
This is not what broke Author filtering. I have described the cause in the description above.